src/cli.js

```diff
@@ -8,6 +10,10 @@ const run = () => {
 	const argv = yargs
 		.commandDir('commands') // commands are in the './commands/' dir
 		.demandCommand() // require a command to be used - no default behavior
+		.option('config', {
```

Comment:
The problem I was trying to solve was "how do i make the package.json config content available to all commands?" There does not appear to be a way to do this _without_ making it some sort of command option. No, I do not see a use case for changing this when executing the command via the CLI.

It seemed like a nice convenience to me. Perhaps others have different views.